spec/jobs/bcc_post_spec.rb

```diff
@@ -54,5 +56,14 @@
 
       expect(Topic.count).to eq(topic_count + 2)
     end
+    
+    it 'works with username and email personalization' do
+      topic_count = Topic.count
+
+      ::Jobs::BccPost.new.execute(user_id: sender.id, create_params: create_params.merge("raw": "this is the content I want to send to {username}", target_emails: 'test@test.com'))
+      
```

Comment:
This really should check if the username was replaced. Right now it only seems to be checking if a post was made!